bugbug/db.py

```diff
@@ -152,6 +152,10 @@ def read(self):
             pass
 
 
+class LastModifiedNotAvailable(Exception):
```

Comment:
Nit: can you move this to earlier than the `last_modified` function or at the top of the file? I prefer the definitions to be earlier than the usages.

Reply:
Moved to top of file.
Had kept it there so that all class definitions are together.